doc/user/isisd.rst

```diff
@@ -394,7 +394,7 @@ Known limitations:
    clear the Node flag that is set by default for Prefix-SIDs associated to
    loopback addresses. This option is necessary to configure Anycast-SIDs.
 
-.. clicmd:: show isis segment-routing prefix-sids
+.. clicmd:: show isis route prefix-sid
```

Comment:
It's documented – `show isis route [level-1|level-2] [prefix-sid|backup]`.
We should only delete the old one.

Reply:
I did not check this.